data/modern/wizards_of_the_coast/msrd/basics/modern_basics/modernbasics_abilities_class.lst

```diff
@@ -33,11 +33,11 @@ Turn/Rebuke Outsiders		CATEGORY:Special Ability	TYPE:Special.Turning.Outsiders	V
 
 ###Block: FX Class Abilities
 # Ability Name	Category of Ability		Type				Visible	Special ABility				Define			Description																											Add Spell Lvl	Modify VAR			Allow Follower		Allowed Companions										Source Page
-Brew Potion		CATEGORY:Special Ability					VISIBLE:NO							DEFINE:BrewLevel|0	DESC:Brew a potion, level is set at time of creation...																						BONUS:VAR|BrewLevel|3																		SOURCEPAGE:msrdadvancedclassesII
-Maximize Spell	CATEGORY:Special Ability					VISIBLE:NO											DESC:All variable,numeric effects of a spell automatically achieve their maximum values; treat any spell this is a pplied to as three higher for memorization purposes	ADDSPELLLEVEL:3																						SOURCEPAGE:msrdadvancedclassesII
+Brew Potion		CATEGORY:Special Ability					VISIBLE:NO							DEFINE:BrewLevel|0	SAB:Brew a potion, level is set at time of creation...																						BONUS:VAR|BrewLevel|3																		SOURCEPAGE:msrdadvancedclassesII
```

Comment:
Why? This change makes zero sense. SAB is an old tag and not meant to be used as part of an ability. Note: the VISIBLE:NO and lack of TYPE tag indicates nobody wanted the to actually output. If you want to correct this, then change to TYPE:SpecialQuality and remove the VISIBLE:NO tag.